### PR TITLE
8467 new homepage design

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-8467

### CODE REVIEW

File: source/partials/_footer.erb [hunk 1]
```diff
@@ -14,10 +8,16 @@
         <%= link_to 'Why Codio?', '/about/mission' %>
       </li>
       <li>
-        <%= link_to 'Our Team', '/about' %>
+        <%= link_to 'Customers', 'http://news.codio.com' %>
```
Comment (mloginov) [inline source/partials/_footer.erb]: please remove protocol if links goes to the same site

File: source/partials/_footer.erb [hunk 2]
```diff
@@ -28,80 +28,92 @@
...
-        <%= link_to 'Status', 'https://status.codio.com', target: '_blank' %>
+        <%= link_to 'Demo', '/demo' %>
       </li>
       <li>
-        <%= link_to 'Twitter', 'https://twitter.com/CODIOHQ', target: '_blank' %>
+        <%= link_to 'Login', '/p/login' %>
       </li>
       <li>
-        <%= link_to 'Facebook', 'https://www.facebook.com/CodioHQ/', target: '_blank' %>
+        <%= link_to 'Join Class', 'https://staging.codiodev.com/p/join-class' %>
```
Comment (mloginov) [inline source/partials/_footer.erb]: @denpira staging??

File: source/partials/_nav.erb [hunk 3]
```diff
@@ -1,29 +1,12 @@
...
-  <li <% if current_page.path.start_with?("demo") then concat 'class="active"' end %>>
-    <%= link_to 'Demo', '/demo', class: 'button' %>
-  </li>
-  <li class='highspot'>
-    <%= link_to 'Join Class', '/p/join-class', class: 'button' %>
-  </li>
-  <li class='highspot'>
-    <%= link_to 'Free trial', (defined?(surl) && !surl.nil? ? surl : '/start-edu-trial'), class: 'button' %>
+  <li>
+    <%= link_to 'Join Class', 'https://staging.codiodev.com/p/join-class' %>
```
Comment (mloginov) [inline source/partials/_nav.erb]: staging

File: source/partials/_header.erb [hunk 4]
```diff
@@ -4,12 +4,27 @@
...
+<header class="fixed">
   <div class="container animated fadeInDown">
-    <a href="/" class="logo-link"><%= image_tag 'logo-light.svg', alt: 'Codio', class: 'logo' %></a>
+  	<div class="logo">
+			<a class="menu-right-toggle fixed open dropdown-toggle" id="header-menu-dropdown"
+				data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+  			<div class="menubars"></div>
+			</a>
+			<div class="dropdown-menu header-dropdown-menu" aria-labelledby="header-menu-dropdown">
+    		<a class="dropdown-item" href="#">School Leadership</a>
```
Comment (mloginov) [inline source/partials/_header.erb]: this link does nothing? is it ok?

Comment (denpira) [inline source/partials/_header.erb]: These pages are absent yet. They will appear later.

**Comment by MaximKraev** [comment]
merged in https://github.com/codio/site/issues/239

**Pull request closed without merging**